database/database.go

```diff
@@ -0,0 +1,91 @@
+package database
+
+import (
+	"dashboard/model"
+	"database/sql"
+	"log"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+var db *sql.DB
+var err error
+
+//InitialiseDb to start db server
+func InitialiseDb() {
+	db, err = sql.Open("mysql", "root:mysql1234@tcp(localhost:3306)/dashboard")
+	if err != nil {
+		log.Panic(err.Error())
+	}
+}
+
+//CloseDb to close DB connection
+func CloseDb() {
+	db.Close()
+}
+
+//InsertUser func for sign_up
+func InsertUser(user *model.UserProfile) string {
+
+	stmtOut, err := db.Prepare("SELECT username FROM users WHERE username  = ? ")

```

Comment:
Refactor this function into two separate functions GetUserProfileByUsername and insertUserProfile functions.